service/pure/src/pure.rs

```diff
@@ -55,8 +54,18 @@ fn save_external_api_multiaddr(multiaddr: Multiaddr) {
     write_config(config);
 }
 
-fn load_external_api_multiaddr() -> eyre::Result<Multiaddr> {
-    load_config().external_api_multiaddr.ok_or(eyre::eyre!("multiaddr is not set"))
+fn load_external_api_multiaddr() -> eyre::Result<String> {
+    load_config().external_api_multiaddr.map(|m| m.to_string()).ok_or(eyre::eyre!("external api multiaddr is not set"))
```

Comment:
why doing `to_string` here? Sometimes you do `load_external_api_multiaddr().is_ok()` -- no reason to always convert to string